src/cobra/test/test_util/test_context.py

```diff
@@ -0,0 +1,52 @@
+"""Test functions of context.py."""
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+from cobra.util import HistoryManager, get_context, resettable
+
+
+if TYPE_CHECKING:
+    from cobra import Model
```

Comment:
Just as an informative comment. I think it would be okay to directly import in the tests. Once they're moved out of the source tree, there cannot be any circular references.

Reply:
We will need to revamp the test suite as a whole when we move it out. I think we can handle it then.